### DISCUSSION TITLE
Not able to commission lightbulb_color example code

### DISCUSSION THREAD

**Discussion by yogeshmiggalore-eaton** [SiliconLabs/arduino#43] [Q&A]
I followed the following steps for matter_lightbulb.
1. Erased the flash chip using Simplicity Studio. ( to erase previously stored commission data )
2. update bootloader code using burn bootloader.
3. programmed matter_lightbulb example code.
4. commissioned matter device using alexa mobile app and echo 4th gen.
5. Tested with alexa mobile app and echo, worked as expected. 

for matter_lightbulb_color example
1. Followed step 1 and step 2.
2. programmed matter_lightbulb_color example code.
3. commissioned failed, in alexa app it is asking to select thread border router to commission but in the device serial print says "device commissioned".

Not able to understand the issue. repeated above mentioned steps multiple times. same result.

**Comment by silabs-bozont** [comment]
Hi,
While the commissioning failed the second time your board may already have saved some data in it's NVM - so it thinks it's been commissioned - while your Alexa thinks it's not.
I'd advise to erase the board again and try the whole commissioning process from the start.
Let me know if it worked!

**Reply by lboue** [reply]
@yogeshmiggalore-eaton Did it work?
Version 2.0.0 was released today.

**Reply by silabs-bozont** [reply]
@yogeshmiggalore-eaton please try it again with the updated (2.0.0) core, and let me know if it solves your issue!
